state_machine/states/impl/land_impl.py

```diff
@@ -30,22 +34,18 @@ async def run(self: Land) -> None:
         update_state("Land")
         update_drone(self.drone)
         update_flight_settings(self.flight_settings)
-
-        logging.info("Landing")
+        logging.info("Land state running")
 
         # Instruct the drone to land
-        await self.drone.system.action.return_to_launch()
-
-        await asyncio.sleep(5)
-        async for flight_mode in self.drone.system.telemetry.flight_mode():
-            if flight_mode != FlightMode.RETURN_TO_LAUNCH:
-                break
-            await asyncio.sleep(1)
-
-        try:
-            await self.drone.system.action.disarm()
-        except ActionError:
-            logging.warning("Unable to disarm, may already be disarmed.")
+        self.drone.vehicle.mode = dronekit.VehicleMode("RTL")
```

Comment:
Use return to launch command in drone.py. Would be Nice to use it in unit tests also, but not necessary